File: PAC/JS/Favourite.js
//$(document).ready(function () {
//    //var grid = $find("<%=ResultsRadgrid.ClientID%>");
//    //var MasterTable = grid.get_masterTableView();
//    //var likebtn = MasterTable.$find("<%=testbtn.ClientID%>");
//    $('#testbtn').click(function () {
//        var favourite = {};
//        var lbmember = $('<%memberid.ClientID%>').text();
//        var lbanimal = $(".well").text();
//        favourite.MemberListID = lbmember;
//        favourite.AnimalListID = lbanimal;
//        $.ajax({
//            url: 'FavouriteService.asmx/AddFavourite',
//            method: 'post',
//            contentType: 'application/json;charset=utf-8',
//            data: '{favourite:' + JSON.stringify(favourite) + '}'
//        })

//    });
//});

File: PAC/Favourite.cs
using System;
using System.Web;

namespace PAC
{
    public class Favourite
    {

        public Favourite(string memid, string anid, string stat)
        {
            MemberListID = memid;
            AnimalListID = anid;
            RecordStatus = stat;
        }

        public Favourite() { }
        public string MemberListID { get; set; }
        public string AnimalListID { get; set; }
        public string RecordStatus { get; set; }

    }
}
